zkSNACKs/WalletWasabi
# Start Wasabi on macOS login

Part of #5956 

On macOS we need to add `Wasabi Wallet.app` to the `Login Items` of the `Current User` in order to start the app on login.
When the user makes the change (wants to start WW on startup), we need to run/use an Applescript.

**How to test:**

1. Get a WW backport release, we need to have a `Wallet Wasabi.app`.
2. Get this PR too.
3. In `General Settings` fiddle with the `Start Wasabi with the system` option.
4. See in the `Login Items` that `WasabiWallet` appeared/got deleted. 
(`Apple menu`  > `System Preferences`, then click `Users & Groups`, select your `user account`, then click `Login Items` at the top of the window.)

ToDo:
- [x] Double check the Pr when it's ready for review
